contao/languages/en/tl_metamodel_dcasetting.php

```diff
@@ -58,15 +58,15 @@
 $GLOBALS['TL_LANG']['tl_metamodel_dcasetting']['searchable'][0]              = 'Searchable';
 $GLOBALS['TL_LANG']['tl_metamodel_dcasetting']['searchable'][1]              =
     'Check if this attribute shall be available for backend search.';
-$GLOBALS['TL_LANG']['tl_metamodel_dcasetting']['allowHtml'][0]               = 'Do not strip html content.';
+$GLOBALS['TL_LANG']['tl_metamodel_dcasetting']['allowHtml'][0]               = 'Do not encode allowed html tags.';
 $GLOBALS['TL_LANG']['tl_metamodel_dcasetting']['allowHtml'][1]               =
-    'If you select this, HTML content will be preserved.';
-$GLOBALS['TL_LANG']['tl_metamodel_dcasetting']['preserveTags'][0]            = 'Do not encode html tags.';
+    'If you select this, allowed HTML tags from system settings will not be encoded.';
+$GLOBALS['TL_LANG']['tl_metamodel_dcasetting']['preserveTags'][0]            = 'Do not encode all html tags.';
 $GLOBALS['TL_LANG']['tl_metamodel_dcasetting']['preserveTags'][1]            =
-    'If you select this, HTML tags will not be encoded.';
+    'If you select this, all HTML tags will not be encoded.';
```

Comment:
Better phrasing: `If you select this, no HTML tags will be encoded.`

Reply:
fixed